libs/components/flyout/src/lib/modules/flyout/flyout.component.scss

```diff
@@ -232,139 +163,13 @@
       var(--sky-space-gap-action_group-m)
     );
     display: inline-block;
-
-    &.sky-btn-link-inline {
-      // These padding, line-height, and vertical align rules can be removed when all that is left is modern v2;
```

Comment:
are we super confident about all this removed stuff? "when only modern v2 is left" makes me think when both default and modern v1 have gone away, not just modern v1